packages/typescript/package.json

```diff
@@ -1,14 +1,21 @@
 {
   "name": "@frontside/typescript",
-  "version": "1.0.1",
+  "version": "1.1.0",
   "description": "TypeScript Development Tools at Frontside",
   "author": "Frontside Engineering <engineering@frontside.com>",
   "dependencies": {
-    "@frontside/tsconfig": "1.0.0",
-    "eslint": "4.18.2",
-    "typescript": "^3.9.7"
+    "@frontside/eslint-config": "1.2.0",
+    "@frontside/tsconfig": "1.1.0",
+    "@typescript-eslint/eslint-plugin": "^3.10.1",
+    "@typescript-eslint/parser": "^3.10.1",
```

Comment:
As Charles said, extending `frontside/eslint-config` is a whole different thing. We were working on a workaround by creating custom commands with `frontside/typescript`. So for now you can omit `@frontside/eslint-config`, `@typescript-eslint/eslint-plugin`, and `@typescript-eslint/parser`.

Reply:
removed